src/components/SideNavigation/NavigationItem/index.tsx

```diff
@@ -22,6 +22,13 @@ const NavigationItem = (props: TNavigationLinkPropTypes) => {
 
     const [childOpen, setChildOpen] = useState(false)
 
```

Comment:
I think here we just need to check isOpen

if(!isOpen){ return null}
return As()

Reply:
@LilitArz jan we have 2 state . 1 navigation isOpen or closed ( this state provide only styles ) and second is childOpen or not , and I need second state for understand when open child accordion

Reply:
Comment is about `displayNavigationItem` fn

Reply:
I need few checking, I can't check only isOpen or not 
![Uploading image.png…]()

Reply:
![photo_2023-12-18_12-15-49](https://github.com/dinofrontend/dino_ui_library/assets/41949946/ae17fe86-d5cf-499b-9c9e-4b8424c9df10)
 I can't check only isOpen or not, I have few checking

Reply:
@LilitArz jan done